aiken-verifier/aiken_halo2/aiken.lock

```diff
@@ -24,4 +24,4 @@ requirements = []
 source = "github"
```

Comment:
can we put aiken.lock under gitignore? it's constantly changed and causing troubles to pull